CONTRIBUTING.rst

```diff
@@ -67,11 +67,10 @@ With respect to code standards, we follow `PEP8`_ and the `Google Style Guide`_.
 .. _`Google Style Guide` : https://google.github.io/styleguide/pyguide.html
 
 
-Licensing
+Licensing of contributions
 --------
 
-The License of this project is located in `LICENSE`_. By submitting a contribution to this project, you are agreeing that your contributions under this license and
-with this waiver of copyright interest.
+By submitting a contribution to this project, you are granting the project maintainers the ability to release your contribution under the project license located in `LICENSE`_. Along with this, you also waiver copyright interest.
```

Comment:
Hmm... What does 'you also waive~~r~~ copyright interest' mean? I think just the first sentence is sufficient.